chef_master/source/troubleshooting_chef_automate.rst

```diff
@@ -364,3 +364,10 @@ You can work around this bug by issuing the following commands on the Chef Serve
    chef-server-ctl restart nginx
 
 It's also recommended that you configure that same token in ``/etc/opscode/chef-server.rb``, and then run ``chef-server-ctl reconfigure``. This will allow you to confirm that the correct token is used to access the Automate system.
+
+Issue: Nodes tab does not show information beyond 100 pages
+------------------------------------------------------------------------------
+
+In the Nodes tab, there is a bug that occurs when the node list exceeds 100 pages (10,000 nodes). As a result of this bug, pages beyond the first 100 will not provide node data.
```

Comment:
This is correct, but also a bit vague. The user will see all 100 pages in the pagination, but when clicking beyond page 100 the results will fail to update; any page for page 100 or less will update correctly.